clang/lib/AST/ASTDiagnostic.cpp

```diff
@@ -1371,11 +1371,17 @@ class TemplateDiff {
   /// argument info into a tree.
   void DiffTemplate(const TemplateSpecializationType *FromTST,
                     const TemplateSpecializationType *ToTST) {
+    // FIXME: With P3310R0, A TST formed from a DeducedTemplateName might
+    // differ in template arguments which were not written.
```

Comment:
I am not sure we need that fixme now

Reply:
It's still possible these types might appear in diagnostics, even if they don't make it to the final program.